src/Type/Definition/EnumType.php

```diff
@@ -147,7 +147,7 @@ public function serialize($value)
             return $lookup[$value]->name;
         }
 
-        throw new Error('Cannot serialize value as enum: ' . Utils::printSafe($value));
+        throw new InvariantViolation('Cannot serialize value as enum: ' . Utils::printSafe($value));
```

Comment:
Just like in scalar types, when serializing a faulty internal value, it can be unsafe to show it in the result. The client won't be able to do much about it anyways.